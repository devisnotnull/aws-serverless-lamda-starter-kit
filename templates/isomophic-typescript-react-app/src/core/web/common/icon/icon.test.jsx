import React from 'react';
import { shallow } from 'enzyme';

import SvgIcon from './icon';

describe('SvgIcon />', () => {
  it('snapshot: renders basket icon', () => {
    const wrapper = shallow(<SvgIcon name="basket" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders locale icon', () => {
    const wrapper = shallow(<SvgIcon name="locale" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick icon', () => {
    const wrapper = shallow(<SvgIcon name="tick" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick arrow', () => {
    const wrapper = shallow(<SvgIcon name="arrow" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick arrow-right', () => {
    const wrapper = shallow(<SvgIcon name="arrow-right" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick arrow-down', () => {
    const wrapper = shallow(<SvgIcon name="arrow-down" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick arrow-left', () => {
    const wrapper = shallow(<SvgIcon name="arrow-left" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('snapshot: renders tick arrow-up', () => {
    const wrapper = shallow(<SvgIcon name="arrow-up" />);
    expect(wrapper).toMatchSnapshot();
  });
});