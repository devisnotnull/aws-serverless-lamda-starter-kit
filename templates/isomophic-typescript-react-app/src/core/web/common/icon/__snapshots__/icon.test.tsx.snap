// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon /> snapshot: renders basket icon 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders locale icon 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick arrow 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick arrow-down 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick arrow-left 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick arrow-right 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick arrow-up 1`] = `ShallowWrapper {}`;

exports[`SvgIcon /> snapshot: renders tick icon 1`] = `ShallowWrapper {}`;
